salvo/tools/typecheck.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```

Comment:
(fyi) Sorry to see that pytype_binary() and pytype_library() bazel targets aren't available in OSS yet.

https://github.com/bazelbuild/bazel/issues/4948

Thank you for figuring out how to perform type checks without them. Do we want to file an issue talking about adding this script to the CI so that we can track it?

Reply:
Certainly.  I added https://github.com/envoyproxy/envoy-perf/issues/76 to track this.